### PR TITLE
fix: Fix provider for iks

### PR DESCRIPTION
**Pull request by ctolon22**
### Description

Add CA certificate to the provider to avoid certificate issue with IKS clusters.

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Review by ocofaigh — CHANGES_REQUESTED** [review]
You need to run the pre-commit hook to correctly format the change otherwise it will fail pipeline. Also do you know if its needed for helm provider too?

**Comment by ctolon22** [comment]
@ocofaigh you're right. It was required in Helm only. I have done an additional test.
I have also run pre-commit hooks.

It should be ok now

File: tests/go.mod [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module github.com/terraform-ibm-modules/terraform-ibm-scc-da
```
Comment (ocofaigh) [inline tests/go.mod:1]: @ctolon22 Can you change this back?

Comment (ctolon22) [reply tests/go.mod]: sorry, i had a different name in the fork producing this change in the pre-commit hook and missed it in the commit. Hope is ok now

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**